xAPI.md

```diff
@@ -255,6 +255,25 @@ learning that SCORM could not enable.
 
 ## 1.0 Summary of the Experience API
 
+The Experience API (Application Programming Interface) - or xAPI as it is commonly called - is a framework that 
+facilitates the description of learning experiences and their communication and storage in a consistent manner 
+in a digital environment. As an application programming interface, the xAPI does not set out how the framework  
+should be implemented. Rather, it sets out the essential components of that framework and how they relate to each other. The purpose of the xAPI and the goals of this document are:
+
+* To maximize interoperability and reliability of computer programs which create, gather and store information  
+about learning experiences.
+* To provide a guide to computer programmers who want to build applications that conform to and implement this 
+framework.
+* To provide users of these programs a description of features and capabilities that they can expect from them.
+* To provide the ADL that certifies conformance to this framework with criteria against which certification can 
+be tested.
+
+There are three main ways that the xAPI promotes the interoperability and reliability of programs that implement the framework. The first is by requiring that programs that gather and store information about learning experiences describe those experiences in a consistent manner. To that end, it sets out a set of terms for describing those experiences including such terms as "Actor", "Activity", "Verb", "Result", and "Context" as well as rules of syntax for how they should be combined. 
+
+The second way that the xAPI promotes interoperability is by setting out the transfer methods that must be used when communicating information about learning experiences between programs that adhere to the framework. As part of this, it sets out the information that needs to be included and how it must be handled via RESTFul HTTP methods. 
```

Comment:
:+1: for 

> must be handled via RESTFul HTTP methods.

(although should be "RESTful", minor typo)

I really think we can do a "REST audit", leaving the current not-quite-RESTful methods intact, e.g. `https://my-lrs.org/xapi/statements?statementId=some-long-uuid` instead of `/statements/some-long-uuid`, etc. - have both for the time being, make the latter a SHOULD*, and deprecate the former in a future version according to semver adherence.

It's really not just about theoretical purity, it's about ease of adoption - if I'm building an AP or reporting tool with a properly opinionated REST client, the latter is going to save me time an headaches.

I have limited time and abhor headaches :smiley_cat: